deps/Chakra.ICU/Chakra.ICU.Common.vcxproj

```diff
@@ -29,6 +29,7 @@
     </ClCompile>
     <Link Condition="'$(ChakraICU)'=='shared'">
       <SubSystem>Console</SubSystem>
+      <AdditionalDependencies>%(AdditionalDependencies);advapi32.lib</AdditionalDependencies>
```

Comment:
This is for InitializeSecurityDescriptor, SetSecurityDescriptorDacl, and a handful of other functions. It seems to only actually be required for ARM, but common_uwp doesn't link against advapi32 on ARM. Are these APIs covered by WindowsApp.lib or one of the other dependencies in common_uwp?

Reply:
Can you scope this dependencies only to ARM then?